README.md

```diff
@@ -69,7 +69,9 @@ By default this interface will ask the OLS server to retain and use your convers
 ### In Cluster Deployment
 Deploying OLS on an openshift cluster is fairly easy with the configuration files we have in [config](./config) folder.
 
-Initial step would be to build and publish our own image to an image registry. Example:
+You can use the existing image built from the latest code: quay.io/openshift/lightspeed-service-api:latest
+
+If you need to build your own image, you can use the following commands:
 
 ```
 podman build -f Containerfile -t=quay.io/<your-repo>/ols:latest .
```

Comment:
I think we need to keep this same as line #83 for consistency.

Reply:
i meant to change it to:
export OLS_IMAGE=<image-pullspec>

and i'm going to change the others also (no reason to assume they are using quay)

Reply:
updated